### PR TITLE
[Project Submission] Parametric Digital Asset Risk Management

### PR DESCRIPTION
**Pull request by cenobyte321**
Adding Parametric Digital Asset Risk Management project submission

### CODE REVIEW

File: projects/pdarm.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+## Use of IPFS, Filecoin and Libp2p
+
+We used IPFS to deploy and host our project’s application, website, store and display images on the website, and to display smart contract analytical data on user dashboards.  
+
+## Project Status
+
+Under development  
+
+## Previews
```
Comment (autonome) [inline projects/pdarm.md:35]: please add a screenshot here

File: projects/pdarm.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+4
+
+### Team members
+
+-   Seth Andrus: System Architecture
+-   Luis Ramírez: Smart Contract & Frontend Development
+-   Mike Robinson: Product Manager & Market Researcher
+-   Lukas Sexton: UI/UX design and research
+
+## How the community can engage
```
Comment (autonome) [inline projects/pdarm.md:78]: can you create a discussion thread in the ipfs/community discussion board and add link in this section? instructions here: https://github.com/ipfs/community#ecosystem-projects

**Review by autonome — CHANGES_REQUESTED** [review]

File: projects/pdarm.md (hunk 1 above)
Comment (cenobyte321) [reply projects/pdarm.md:35]: Screenshots added

File: projects/pdarm.md (hunk 2 above)
Comment (cenobyte321) [reply projects/pdarm.md:84]: Sure. I have created the discussion thread and added the link here.

**Comment by autonome** [comment]
Looks good, thanks!

**Pull request merged**